repos/elucidate/view/elucidate/activity/user-activity.twig

```diff
@@ -17,17 +19,17 @@
                 {% for annotation in annotationsSortable.onlyCanvas(canvas.canvasSource.omekaUri).get() %}
                     <div class="c-dig-grid">
                         <div class="c-dig-grid__col--2">
-                            {# @todo timeAgo to be added #}
                             {{ annotation.timeAgo }}
                         </div>
                         <div class="c-dig-grid__col--10">
                             {# @todo motiviation to be added #}
                             {% if(annotation.label or canvas.label) %}
-                                Added a <strong>{{ annotation.motivation }}</strong> annotation to
+                                {{ translate('Added a %s annotation to') | format(annotation.motivation) }}
```

Comment:
Hm, just noticed we give the wrong inflection here for motivations starting with a vowel. Not sure how you could translate that, though. LGTM

Reply:
Yeah Annotation motivations could be any, Ill maybe switch it over to `added an annotation (%s) to`